### DISCUSSION TITLE
SuperSlicer Translation

### DISCUSSION THREAD

**Discussion by SrBogoePT** [supermerill/SuperSlicer#2293] [Community contribution]
I would like to know if it is possible to have a translation from SuperSlicer to 100% Portuguese?, since some things are in Portuguese and English all together, and that each translation should be 100% translated into your language, if possible I'm available to help in translating it, just ask.

Compliments
Arthur Jesus

**Comment by supermerill** [comment]
yes.
See https://github.com/supermerill/SuperSlicer/issues/2227 for a quick startup, and the readme in settings/localization for the doc.
